ERCS/erc-7695.md

```diff
@@ -0,0 +1,477 @@
+---
+eip: 7695
+title: Ownership Delegation and Context for ERC-721
+description: Extends ERC-721 for financial use cases without changing owner
```

Comment:
Your description is a bit vague. Think you could expand it a bit?

Maybe something about creating different contexts, and delegating permissions within those contexts, without changing the ultimate owner of the token?

Reply:
It was changed to _"Introduces contexts and ownership delegation for ERC-721 tokens, expanding dApps and financial use cases without transferring ownership"_ in this commit https://github.com/ethereum/ERCs/pull/391/commits/064e1e7b60cb8275546c5c2649271a3a7428a9dc
Let me know if I can make it better. Thank you.